components/server/src/workspace/workspace-starter.ts

```diff
@@ -1435,17 +1436,23 @@ export class WorkspaceStarter {
             }
         }
 
-        let volumeSnapshotId = ""
+        let volumeSnapshotId = "";
         // always pick lastValidWorkspaceInstanceId if it is valid, otherwise workspace will be restored from prebuild
         // even if there was workspace backup available
         if (lastValidWorkspaceInstanceId != "") {
-            volumeSnapshotId = lastValidWorkspaceInstanceId
-        } else if ((SnapshotContext.is(workspace.context) || WithPrebuild.is(workspace.context)) && !!workspace.context.snapshotBucketId) {
-            volumeSnapshotId = workspace.context.snapshotBucketId
+            volumeSnapshotId = lastValidWorkspaceInstanceId;
+        } else if (
+            (SnapshotContext.is(workspace.context) || WithPrebuild.is(workspace.context)) &&
+            !!workspace.context.snapshotBucketId
+        ) {
+            volumeSnapshotId = workspace.context.snapshotBucketId;
```

Comment:
This changes behavior, and looks like a rebase error. @AlexTugarev can you confirm?

Reply:
@geropl, could you clarify, please. 
Cannot see any significant changes. I touched the `System` param in that file, and it got formatted on save. Formatter changes look ok to me.

Reply:
@AlexTugarev Sorry, thought I'd seen an additional `else` branch here. :see_no_evil: Definitely gone now. :stuck_out_tongue: 
Sorry for the noise!

Reply:
No problemo 👍🏻